src/workflows/workflows_test.ts

```diff
@@ -173,3 +173,46 @@ Deno.test("Workflows properly treats interactivity and user context types", () =
     `{{steps.0.user.secret}}`,
   );
 });
+
+/*
+  Deno will exit on uncaught exceptions. 
+  JSON.parse will raise an exception when given undefined. 
+  By setting undefined manually to reflect the case of an 
+  undefined inputValue, which will allow Deno to continue 
+  and pass the undefined values up to the validation API 
+  -- which will then communicate back to the CLI the specific 
+  validation errors it ran in to.
+*/
```

Comment:
@shapirone 👀 🤔

Reply:
Sorry, it looks like the Deno tests are failing because of the multi-line comments: 

<img width="684" alt="Screen Shot 2022-12-13 at 9 52 11 AM" src="https://user-images.githubusercontent.com/4774899/207408219-9ed52654-f4f8-4e77-85e5-ff1b09a9817b.png">

I'll switch it to `//`.

Reply:
That seems to [have worked](https://github.com/slackapi/deno-slack-sdk/actions/runs/3688011788/jobs/6242303353).